resources/light/cdk/cloudformation.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 75 75"><defs><style>.cls-1{fill:#cc2264;}.cls-2{fill:#fff;}</style></defs><title>AWS-CloudFormation_light-bg</title><g id="Reference"><rect id="Pink_Light_BG" data-name="Pink Light BG" class="cls-1" width="75" height="75"/><g id="Product_Icon" data-name="Product Icon"><path class="cls-2" d="M62.5,35A8,8,0,0,1,55,43.12H40.5v-2H55c.14,0,5.55-.6,5.55-6.13,0-4.64-3.88-5.8-5.54-6.09a1,1,0,0,1-.82-1.14c0-2-.84-3.43-2.31-3.91A3.22,3.22,0,0,0,48.23,25a1,1,0,0,1-1,.39,1,1,0,0,1-.79-.65A10.84,10.84,0,0,0,44,20.65a8.94,8.94,0,0,0-10.1-2,9,9,0,0,0-4.4,4.47H27.3a11.16,11.16,0,0,1,5.79-6.31,10.94,10.94,0,0,1,12.29,2.42,12.33,12.33,0,0,1,2.4,3.44A5.2,5.2,0,0,1,52.45,22a5.56,5.56,0,0,1,3.66,5.15C59.33,27.92,62.5,30.35,62.5,35Zm-28-.87v-7h-20v30h20v-17h2v18a1,1,0,0,1-1,1h-22a1,1,0,0,1-1-1v-32a1,1,0,0,1,1-1h22a1,1,0,0,1,1,1v8Zm13,2h-30v2h30Zm-22-5h-8v2h8Zm3,10h-11v2h11Zm-4.09,5H17.54v2h6.87Zm7.09,0h-5v2h5Zm22-10h-4v2h4Z"/></g></g></svg>
```

Comment:
It would be inconsistent at a minimum. I'll make the changes to align with the current feel of the toolkit. I think I would just reuse the CloudFormation icon as well